.travis.yml

```diff
@@ -1,18 +1,16 @@
 sudo: required
 language: generic
-compiler:
-- gcc
+dist: bionic
```

Comment:
The builds run in Docker anyway, and all other apps/CEs use xenial as the Travis dist. I think if there's a reason to use bionic let's do it for all others too, if not - it might be better to stick to xenial

Reply:
Looks like the other sample apps and cloud extensions don't specify distro. Let's remove this for DeepRacer and ObjectTracker.